b2share/modules/records/cli.py

```diff
@@ -54,6 +55,14 @@ def check_dois(update):
         check_record_doi(record, update)
 
 
+@b2records.command()
+@with_appcontext
+def expire_embargoes():
```

Comment:
The name of the function could be confusing for some people. They might think that this will expire all embargoes. What about renaming it to `update_expired_embargoes`? That way it matches the celery task name.